config/routes/backend/partial/customer.yaml

```
@@ -0,0 +1,10 @@
+sylius_backend_partial_customer_latest:
```

Why do you prefix it with sylius?

```suggestion
monofony_backend_partial_customer_latest:
```

We have a new brand here :)

---

I follow SyliusResourceBundle routes generator to be consistent.
if i's a Sylius resource, I prefix by "sylius_" and if it's a custom resource I prefix by "app_".